src/main/java/teammates/storage/search/FeedbackResponseCommentSearchDocument.java

```diff
@@ -39,7 +39,7 @@
     private FeedbackQuestionAttributes relatedQuestion;
     private FeedbackSessionAttributes relatedSession;
     private CourseAttributes course;
-    private InstructorAttributes giverAsInstructor; // comment giver
+    private InstructorAttributes giverAsInstructor;
```

Comment:
I left this to be done, when showing name instead of student's email in `search` Page Results.